src/core/client/admin/components/BanModal.tsx

```diff
@@ -309,54 +313,137 @@ const BanModal: FunctionComponent<Props> = ({
           <Form onSubmit={onFormSubmit}>
             {({ handleSubmit, submitError }) => (
               <form onSubmit={handleSubmit}>
-                <HorizontalGutter spacing={3}>
-                  {updateType !== UpdateType.NO_SITES && (
-                    <Localized
-                      id={
-                        viewerIsSingleSiteMod
-                          ? "community-banModal-reject-existing-singleSite"
-                          : rejectExistingCommentsLocalizationId!
-                      }
+                {/* MAIN */}
+                <Flex
+                  direction="column"
+                  className={styles.form}
+                  justifyContent="flex-start"
+                  spacing={3}
+                >
+                  {/* BAN FROM/REJECT COMMENTS */}
```

Comment:
I'm seeing a lot of commented sections to this large file. Would it make sense to break them out into smaller named components?

Reply:
That would probably be a good idea. I can do that, but this is from one of the commits I didn't mean to include. This is for the PR re: fixing the ban modal design